src/index.spec.js

```diff
@@ -0,0 +1,134 @@
+import sinon from 'sinon'
+import mocksHttp from 'node-mocks-http'
+import middleware from './index'
+
+/* eslint-env mocha */
+/* eslint-disable no-unused-expressions */
+
+const getTestResult = (url = '', config) => {
+  const {
+    apis = [],
+    force = false,
+    enabled = false,
+  } = config || {}
+
+  const next = sinon.spy()
+  const req = mocksHttp.createRequest({ url })
+  const res = mocksHttp.createResponse()
+  sinon.spy(res, 'send')
+
+  middleware(apis, { force, enabled })(req, res, next)
+
+  return { next, req, res }
+}
+
+describe('express-mockups-middleware', () => {
+  describe('calling next function', () => {
+    it('should call next when "mockups" argument is not present or "enabled" argument equals false', () => { // eslint-disable-line max-len
```

Comment:
"mockups" and "enabled" are not same kind of arguments.
You could write `"mockups" CLI argument` ?